articles/what-api-endpoints-to-expose-to-the-public-internet.md

```diff
@@ -20,23 +20,51 @@ For full Fleet Desktop and scripts functionality, `/api/fleet/orbit/*` and`/api/
 If you would like to use the fleetctl CLI from outside of your network, the following endpoints will also need to be exposed for `fleetctl`:
 
 - `/api/setup`
-- `/api/v1/setup`
-- `/api/latest/fleet/*`
-- `/api/v1/fleet/*`
+- `/api/*/setup`
+- `/api/*/fleet/*`
 
 ## Using Fleet's MDM features
 
-If you would like to use Fleet's MDM features, the following endpoints need to be exposed:
+### macOS
+
+If you would like to use Fleet's macOS MDM features, the following endpoints need to be exposed:
 
 - `/mdm/apple/scep` to allow hosts to obtain a SCEP certificate.
 - `/mdm/apple/mdm` to allow hosts to reach the server using the MDM protocol.
 - `/api/mdm/apple/enroll` to allow DEP-enrolled devices to get an enrollment profile.
-- `/api/*/fleet/device/*/mdm/apple/manual_enrollment_profile` to allow manually enrolled devices to
-  download an enrollment profile.
+- `/api/*/fleet/device/*` to give end users access to their **My device** page. This page is where they download their manual enrollment profile, rotate their disk encryption key, and use other features. For more information on these API endpoints see the documentation [here](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/API-for-contributors.md#device-authenticated-routes).
+- `/api/*/fleet/mdm/sso` and `/api/*/fleet/mdm/sso/callback` if you use automatic enrollment (DEP) and you require [end user authentication](https://fleetdm.com/docs/using-fleet/mdm-macos-setup-experience#end-user-authentication-and-eula) during out-of-the-box macOS setup.
+- `/api/*/fleet/mdm/setup/eula/*` if you use automatic enrollment (DEP) and you require that the end user agrees to an [End User License Agreement (EULA)](https://fleetdm.com/docs/using-fleet/mdm-macos-setup-experience#end-user-authentication-and-eula) during out-of-the-box macOS setup.
+- `/api/*/fleet/mdm/bootstrap` if you use automatic enrollment (DEP) and you install a [bootstrap package](https://fleetdm.com/docs/using-fleet/mdm-macos-setup-experience#bootstrap-package) during out-of-the-box macOS setup.
 
 > The `/mdm/apple/scep` and `/mdm/apple/mdm` endpoints are outside of the `/api` path because they
 > are not RESTful and are not intended for use by API clients or browsers.
 
+### Windows
+
+If you would like to use Fleet's Windows MDM features, the following endpoints need to be exposed:
+
+- `/api/mdm/microsoft/discovery` TODO
+- `/api/mdm/microsoft/auth` TODO
+- `/api/mdm/microsoft/policy` TODO
+- `/api/mdm/microsoft/enroll` TODO
+- `/api/mdm/microsoft/management` TODO
```

Comment:
Hey @roperzh I'm trying to briefly describe what each endpoint is used for. 

When you get the chance, can you please help me by taking a crack at the descriptions?

For customers, I think it's most important to know which endpoints are required for automatic enrollment. That way, if they choose to only use manual enrollment, they don't need to expose these endpoints.

Reply:
Hey @roperzh just giving you an extra ping here.

When you get the chance, can you please help me with the descriptions for these endpoints? What do they do and which are required for automatic v. manual enrollment?

Reply:
@noahtalerman thanks for the ping, and sorry for missing this.

I pushed a commit with my best effort at this, feel free to edit and ask if something is not clear.

Reply:
Thank you!

I made some tweaks to the language to make them consistent with other items:

- Start with verb (ex. "allows")
- If the endpoint is for automatic enrollment only. Call that out first
- Break out a new line for more details